app/views/users/show.html.slim

```diff
@@ -117,7 +117,7 @@
                     th scope='col'= '#'
                     th scope='col'= 'Datum'
                     th scope='col'= 'Beschrijving'
-                    th scope='col'= 'Bedrag'
+                    th.text-right scope='col'= 'Bedrag'
```

Comment:
The amount column of orders was also aligned to the right. This looks nicer in my opinion so I also aligned the amount column of credit mutations to the right.